packages/framework/aqueduct/src/request-handlers/requestHandlers.ts

```diff
@@ -65,3 +65,26 @@ export const defaultRouteRequestHandler = (defaultRootId: string) => {
         return undefined; // continue search
     };
 };
+
+/**
+ * Default request handler for a Fluid object that returns the object itself if:
+ *  1. the request url is a "/"
+ *  2. the request url is empty
+ * Returns a 404 error for any other request.
+ */
+export function defaultFluidObjectRequestHandler(fluidObject: IFluidObject, request: IRequest): IResponse {
+    const pathParts = RequestParser.getPathParts(request.url);
```

Comment:
I'd just create RequestParser - there is a code in there that detects and optimizing parsing if passed request object is of RequestParser type.